### DISCUSSION TITLE
LIS Configuration Settings

### DISCUSSION THREAD

**Discussion by dfrancis827** [NASA-LIS/LISF#1107] [General]
I have successfully ran the LIS tutorial and am now working on adapting it to run over a different part of the contiguous USA and different time frame. However, I intend to essentially do the same modeling that was done in the tutorial (i.e., using NOAH 3.6 LSM and assimilating SMAP L3 surface moisture data). With that being said, there is one thing that I am not confident about, and that is what needs to be changed to accurately run LIS over my new area and time frame. From running the tutorial, I know that there are configuration files for the LDT, LVT, and LIS runs. Additionally there is meteorological forcing data that would need to be changed for a LIS run over a new area/time frame. However, aside from changing the configuration files and forcing data, does data in the "INPUT" folder need to be adjusted, or is that default data that remains constant no matter where LIS is ran? 

Summary of question: Configuration files and forcing data need to be adjusted for use when running LIS over an area/time frame different than that of the tutorial test cases. Do files within the "INPUT" folder need to be adjusted/changed or are those meant to be left as they are?

**Comment by karsenau** [comment]
Hi @dfrancis827:

Thank you for your interest in using LIS for your work.

To run over a new domain, you would first start with LDT and create a new LDT config file (e.g., ldt.config_new), and you would specify a new "Run domain" in that config file, for example:

Map projection of the LIS domain:      latlon
Run domain lower left lat:           25.625
Run domain lower left lon:         -124.125
Run domain upper right lat:          52.875
Run domain upper right lon:         -67.875
Run domain resolution (dx):           0.25
Run domain resolution (dy):           0.25

You can specify the bounding gridcells (represented at the center of the gridcell) and the resolution that you would like to run on.  LIS runs on almost any "latlon" (e.g., Plate-Carree grid coordinate system), but you can also run with Lambert Conformal projection.  Also, make sure to update the name of your output LDT-generated netcdf parameter file, example,

Processed LSM parameter filename:    ./lis_input_new.d01.nc

Then rerun LDT again with your LDT executable and config files:   ./LDT ldt.config_new

If you are using NLDAS2 forcing, you can run anywhere within the Contiguous U.S.  If you are running with other forcing datasets, e.g., GDAS - you can run anywhere in the globe.  You don't need to change forcing dataset options, only dependent on the boundary domains of the forcing.  For downscaling (or upscaling) in relation to your run domain resolution, that would depend on the resolution you select.

For the files originally provided in the INPUT directory, you should be able to leave those datasets alone for now, but you can always add to them later.

Hope this information helps.

**Reply by dfrancis827** [reply]
This most certainly helps. Thank you!
